src/sentry/static/sentry/app/components/sidebar/index.tsx

```diff
@@ -195,8 +195,6 @@ class Sidebar extends React.Component<Props, State> {
       'releases',
       'user-feedback',
       'discover',
-      'discover/queries',
-      'discover/results',
```

Comment:
Looks good. This made sense when discover/queries displayed global selection (which it no longer does).